### DISCUSSION TITLE
好消息

### DISCUSSION THREAD

**Discussion by PandaFiredoge** [Richasy/Bili.Copilot#542] [General]
- Windows App SDK 1.6 终于带来了 Native AOT 支持
- .NET 9 支持 UWP 应用开发

**Comment by Richasy** [comment]
已经尝试对哔哩助理进行了 AOT，但效果不尽如人意，WebView2 完全用不了，且在使用期间崩溃的几率也大幅增加，所以还是有待 winappsdk 进一步完善

**Reply by PandaFiredoge** [reply]
应该要等1.7了，我看隔壁[Files也没上AOT](https://github.com/files-community/Files/pull/16156)
